### PR TITLE
XW-1076 | Implement prerender.io on 10 communities

### PR DESCRIPTION
**Pull request by vforge**
https://wikia-inc.atlassian.net/browse/XW-1076

Uses https://github.com/vforge/hapi-prerender (with patches)

### CODE REVIEW

File: server/config/localSettings.base.js [hunk 1]
```diff
@@ -191,6 +192,12 @@ const localSettings = {
 	mwPreviewSalt: process.env.MW_PREVIEW_SALT,
 	// Special salt for calculating GA userID
 	gaUserSalt: process.env.GA_USERID_SALT,
+	// Prerender.io API Key
+	prerenderApiKey: process.env.PRERENDER_API,
+	// prerendered host lists
+	prerenderHost: [
+		'assassinscreed.wikia.com'
```
Comment (vforge) [inline server/config/localSettings.base.js]: This is temporary / need final list

File: server/package.json [hunk 2]
```diff
@@ -22,6 +22,7 @@
     "handlebars": "4.0.5",
     "hapi": "9.5.1",
     "hapi-i18next": "2.2.0",
+    "hapi-prerender": "vforge/hapi-prerender",
```
Comment (rogatty) [inline server/package.json]: Won't it be downloaded every time you install modules? I think we should use commit hash or release tag here.

Comment (vforge) [inline server/package.json]: We'll add tag when we'll have final list of wikis.

Comment (hakubo) [inline server/package.json]: !

File: server/config/localSettings.base.js (hunk 1 above)
Comment (hakubo) [inline server/config/localSettings.base.js]: mkay

**Comment by hakubo** [comment]
LGTM

**Comment by tomasznapieralski** [comment]
LGTM

**Pull request merged**